### PR TITLE
Feature/docs

### LINKED ISSUE #8: Create customer-ready project readme
Create a readme for the Kentico community that describes:
- Purpose
- Installation
- How to use

 The readme for the Sitecore migration toolkit is a good model.

### PR DESCRIPTION
**Pull request by nathaniel-chadwick**
# Motivation

Which issue does this fix? Fixes #8

## Checklist

- [X] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [X] Tests are passing
- [ ] Docs have been updated (if applicable)
- [X] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

## How to test

Only documentation changes. Please review. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,65 @@
...
+
+- [ASP.NET Core 8.0](https://dotnet.microsoft.com/en-us/download)
+- [Xperience by Kentico](https://docs.kentico.com/changelog)
+- [Progress Sitefinity CMS 14.4](https://www.progress.com/documentation/sitefinity-cms/144/sitefinity-cms-documentation)
+- [XbyK Universal Migration Tool](https://github.com/Kentico/xperience-by-kentico-universal-migration-tool)
+
+## Quick Start
+
+1. Download this package from the repository [Releases](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/releases)
+1. Check XbyK and Sitefinity settings to ensure proper communication. See [Usage Guide](./docs/Usage-Guide.md) for details.
```
Comment (marcel-rbro) [inline README.md:33]: If you still need to read the usage guide to setup the integration, I'd just omit the quick start and keep the "Full instructions" as "Setup instructions"

File: docs/Usage-Guide.md [hunk 2]
```diff
@@ -1,3 +1,172 @@
...
----Organize the Usage Guide into sections using Markdown Headings. This guide should mirror the steps in the Quick Start but with far more details and include any optional steps.---
+# Usage Guide
+
+1. Download the package from the repository [Releases](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/releases)
+
+   - Optionally, clone the git repository and build from the source files.
+   - ` git clone -v "https://github.com/Kentico/xperience-by-kentico-sitecore-migration-tool.git" "{YOUR PATH}"`
+
+2. Modify the console application's configuration file `/examples/Migration.Toolkit.Sitefinity.Console/appsettings.json`
+   - The details for configuration options can be found below.
```
Comment (marcel-rbro) [inline docs/Usage-Guide.md:9]: Below where? Add a link to the heading or number of the step

File: docs/Usage-Guide.md [hunk 3]
```diff
@@ -1,3 +1,172 @@
...
+3. Modify code for any additional customizations
+   - Entity Framework is included. Add additional custom models to Migration.Toolkit.Sitefinity.Data.Core.EF.SitefinityContext for any custom tables. An additional import service will need to be created. See other import services.
```
Comment (marcel-rbro) [inline docs/Usage-Guide.md:11]: See other import services where?

File: docs/Usage-Guide.md [hunk 4]
```diff
@@ -1,3 +1,172 @@
...
+   - Use the RestSdkBase when creating a provider that calls the rest api endpoints. This will ensure the rest client is running and provides methods to call the rest client.
+   - Add additional field types for custom field types. See '/src/Migration.Toolkit.Sitefinity/FieldTypes' for OOB field types in Sitefinity. Using the IFieldType, will automatically add it to the import processing. You can also use the FieldTypeBase to handle the defaults.
+4. Set up Sitefinity
+   - Ensure 'Deployment Mode' is set to 'source'. This feature will export the content type definitions from the database and store them on the disk. Like XbyK's Continuous Integration (CI) files. ![alt text](image-5.png)
```
Comment (marcel-rbro) [inline docs/Usage-Guide.md:15]: If you end a line with two spaces, a newline is inserted. Just a tip.

File: docs/Usage-Guide.md [hunk 5]
```diff
@@ -1,3 +1,172 @@
...
+
+```
+{
+  "TypeName": "NewsItem",
+  "PageRootPath": "/news",
+  "PageTemplateType": "Listing"
+}
+```
+
+Sitefinity also allows the user to use the same widget as a detail view. This allows the content editor to select a particular item and render the content as if the structured content was part of the page itself. If the item being shown also had child items, they would typically render in a listing as part of the content. To import a content item as a detail page and the structured content to be associated with that page, the detail template type will need to be added to the configuration. For example, a user may have a page named "/blog-one" that contains the blog detail widget with the "/blog-1" content item selected. The page would render the content for "/blog-1" and show any child blog items as a listing. When navigating to those blog items, the "/blog-one" url will be prepended. For example, "/blog-one/blog-item-1". To mimic this behavior in XbyK, you would add the following record.
```
Comment (marcel-rbro) [inline docs/Usage-Guide.md:125]: "Prepended" is a valid word, but I'd replace it with something more understandable or reword the sentence.

File: README.md (hunk 1 above)
Comment (nathaniel-chadwick) [reply README.md]: This has been resolved. I have removed the Quick Start and updated Full Instructions to Setup Instructions

File: docs/Usage-Guide.md (hunk 2 above)
Comment (nathaniel-chadwick) [reply docs/Usage-Guide.md]: This has been resolved. I have added a link to the "Configuration File Options" section below

File: docs/Usage-Guide.md (hunk 3 above)
Comment (nathaniel-chadwick) [reply docs/Usage-Guide.md]: This has been resolved. I have added a path to the other import services so the reader can check the code.

File: docs/Usage-Guide.md (hunk 4 above)
Comment (nathaniel-chadwick) [reply docs/Usage-Guide.md:15]: Good to know thank you!

File: docs/Usage-Guide.md (hunk 5 above)
Comment (nathaniel-chadwick) [reply docs/Usage-Guide.md]: I have updated the sentence. Please let me know if the rewording resolves the issue. Thank you!

File: docs/Usage-Guide.md [hunk 6]
```diff
@@ -1,3 +1,177 @@
...
+     ![alt text](image-6.png)
+   - Check the default language for source Sitefinity site, it must match what is in XbyK. Look in _Administration >> Settings >> Basic >> Language_!  
+     ![alt text](image-2.png)
+   - Make sure the Sitefinity content API is set to allow all content and is enabled. View the web service configuration under _Administration >> Web services >> Default_  
+     ![alt text](image-4.png)  
+     In the configuration options for 'Default' make sure all content types are selected and 'This service is active' is checked/true.  
+      ![alt text](image-1.png)
+5. Set up XbyK
+   - Modify the XbyK default language to match the default language of the Sitefinity source site being migrated e.g. 'en-us'. Sitefinity's default English culture is 'en'.
```
Comment (marcel-rbro) [inline docs/Usage-Guide.md:27]: What exactly is meant by the default language? I think this should refer to the primary language of a website channel: https://docs.kentico.com/developers-and-admins/configuration/website-channel-management#create-website-channels

File: docs/Usage-Guide.md (hunk 5 above)
Comment (marcel-rbro) [reply docs/Usage-Guide.md]: Better now :)

File: docs/Usage-Guide.md (hunk 6 above)
Comment (nathaniel-chadwick) [reply docs/Usage-Guide.md:28]: The default language is describing the English language in the Languages application in XbyK. The English language is there by default on a new install. The primary language of the channel is handled on import.

Comment (marcel-rbro) [reply docs/Usage-Guide.md:28]: The English language is there by default, but any Language can be set as the default language. This should be changed to "Modify the default 'English' language in XbyK..." if you mean the default English language.

Comment (nathaniel-chadwick) [reply docs/Usage-Guide.md]: Ok this has been fixed in latest commit!

**Review by marcel-rbro — APPROVED** [review]
Review OK 👍

**Pull request merged**